examples/LA/003_LA_ORU_R01_NBS_3_hl7_translation_final_obr_flatten_ex_1.hl7

```diff
@@ -0,0 +1,56 @@
+MSH|^~\&|Natus^natus.health.state.la.us^DNS|Louisiana Office of Public Health Laboratory^2.16.840.1.114222.4.3.26.3.1^ISO|Test Department of tests^1.2.840.114350.1.13.286.2.7.2.695071^ISO|^simulated-hospital-id^L,M,N|20240101050000+0000||ORU^R01^ORU_R01|20240101050000_0001|T|2.5.1|||AL|AL|||||LRI_NG_FRN_PROFILE^^2.16.840.1.113883.9.195.3.4^ISO~LRI_NDBS_COMPONENT^^2.16.840.1.113883.9.195.3.6^ISO~LAB_PRN_Component^^2.16.840.1.113883.9.81^ISO~LAB_TO_COMPONENT^^2.16.840.1.113883.9.22^ISO
```

Comment:
If this file is is the resulting output for 003_LA_ORU_R01_NBS_0_initial_obr_flatten_example_1.hl7, we can run the `scripts/setup/update-examples-snapshots.sh` script to automatically create and update all the snapshots. That could be included in a separate PR

Reply:
What does updating snapshots do? Not sure I've ever used the script

Reply:
It's a new script. It iterates over all the files with the `_0_initial_message.hl7` suffix in the `examples/` folder, sends them to a local instance of RS, and then downloads the snapshots at different points of the flow that we care about, including the final delivery. The idea is to keep all the derived files in `examples/` up to date with that script. You can find more info here: https://github.com/CDCgov/trusted-intermediary/tree/main/scripts#setupupdate-examples-snapshotssh. Also, for it to work, you'd need to rename the files you added to have the `_0_initial_message.hl7` suffix

Reply:
K, i'll add that in a separate for the same story but that's a good idea.

Reply:
The sample files were created with the new script in this branch `story/1355/example-file-script`. Once this branch is merged, I'll create another PR for the new branch.